### PR TITLE
Tag missing internals

### PR DESCRIPTION
**Pull request by ogizanagi**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Comment by dunglas** [comment]
:+1:

File: src/Symfony/Component/DomCrawler/Field/ChoiceFormField.php [hunk 1]
```diff
@@ -155,11 +155,11 @@ public function setValue($value)
     /**
      * Adds a choice to the current ones.
      *
-     * This method should only be used internally.
-     *
      * @param \DOMElement $node
      *
      * @throws \LogicException When choice provided is not multiple nor radio
+     *
+     * @internal Should not be used by user-land code.
```
Comment (nicolas-grekas) [inline src/Symfony/Component/DomCrawler/Field/ChoiceFormField.php]: Comment doesn't add anything to the annotation, I'd suggest removing it, same below

**Comment by stof** [comment]
👍

**Comment by xabbuh** [comment]
👍

**Comment by nicolas-grekas** [comment]
Thank you @ogizanagi.

**Pull request merged**